common-pg/src/main/scala/com/socrata/pg/store/LocalRollupInfo.scala

```diff
@@ -5,8 +5,8 @@ import java.security.MessageDigest
 import com.socrata.datacoordinator.id.RollupName
 import com.socrata.datacoordinator.truth.metadata.{CopyInfo, RollupInfo}
 
-class LocalRollupInfo(copyInfo: CopyInfo, name: RollupName, soql: String, val tableName: String)(implicit tag: com.socrata.datacoordinator.truth.metadata.`-impl`.Tag) extends RollupInfo(copyInfo, name, soql, None) {
-  def updateName(newTableName: String) = new LocalRollupInfo(copyInfo, name, soql, newTableName)
+class LocalRollupInfo(copyInfo: CopyInfo, name: RollupName, soql: String, val tableName: String,val systemId: Long = -1L)(implicit tag: com.socrata.datacoordinator.truth.metadata.`-impl`.Tag) extends RollupInfo(copyInfo, name, soql, None) {
```

Comment:
I generally prefer to have an `Option[Long]` over a magic value, though I'm not sure this even needs to be optional in any sense... (see further comments below)

Also I prefer the various kinds of Ids to have types rather than just being bare numbers.